sql/core/src/test/resources/sql-tests/inputs/pgSQL/aggregates_part3.sql

```diff
@@ -0,0 +1,273 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- AGGREGATES [Part 3]
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/aggregates.sql#L352-L605
```

Comment:
`REL_12_BETA1` -> `REL_12_BETA2`?

Reply:
Please check the line numbers too if the file is changed. I hope it's unchanged.

Reply:
AGGREGATES [Part 3] unchanged, AGGREGATES [Part 4] changed:
https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/aggregates.sql#L992
We has added it: https://github.com/wangyum/spark/blob/2a2b2029622746c0e08a8b12a38bd6b1171a6396/sql/core/src/test/resources/sql-tests/inputs/pgSQL/aggregates_part4.sql#L414-L419